WasatchNET/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="12.0.3" targetFramework="net461" />
+  <package id="Newtonsoft.Json" version="13.0.2" targetFramework="net40" />

```

Comment:
Interesting that you had to move to a NEWER version of the package, but okay :-)